_includes/social.html

```diff
@@ -0,0 +1,13 @@
+  {% if site.facebook_username %}<li><a href="https://www.facebook.com/{{ site.facebook_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#facebook"></use></svg> <span class="username">{{ site.facebook_username | escape }}</span></a></li>{% endif %}
+  {% if site.flickr_username %}<li><a href="https://www.flickr.com/photos/{{ site.flickr_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#flickr"></use></svg> <span class="username">{{ site.flickr_username | escape }}</span></a></li>{% endif %}
+  {% if site.github_username %}<li><a href="https://github.com/{{ site.github_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#github"></use></svg> <span class="username">{{ site.github_username | escape }}</span></a></li>{% endif %}
+  {% if site.instagram_username %}<li><a href="https://instagram.com/{{ site.instagram_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#instagram"></use></svg> <span class="username">{{ site.instagram_username | escape }}</span></a></li>{% endif %}
+  {% if site.linkedin_username %}<li><a href="https://www.linkedin.com/in/{{ site.linkedin_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#linkedin"></use></svg> <span class="username">{{ site.linkedin_username | escape }}</span></a></li>{% endif %}
+  {% if site.pinterest_username %}<li><a href="https://www.pinterest.com/{{ site.pinterest_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#pinterest"></use></svg> <span class="username">{{ site.pinterest_username | escape }}</span></a></li>{% endif %}
+  {% if site.twitter_username %}<li><a href="https://www.twitter.com/{{ site.twitter_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#twitter"></use></svg> <span class="username">{{ site.twitter_username | escape }}</span></a></li>{% endif %}
+  {% if site.youtube_username %}<li><a href="https://youtube.com/{{ site.youtube_username | cgi_escape | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#youtube"></use></svg> <span class="username">{{ site.youtube_username | escape }}</span></a></li>{% endif %}
+  {% if site.googleplus_username %}<li><a href="https://plus.google.com/{{ site.googleplus_username | escape }}"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#googleplus"></use></svg> <span class="username">{{ site.googleplus_username | escape }}</span></a></li>{% endif %}
+  {% if site.rss %}<li><a href="{{ site.baseurl }}/feed.xml"><svg class="svg-icon"><use xlink:href="/assets/icons.svg#rss"></use></svg> <span>{{ site.rss | escape }}</span></a></li>{% endif %}
+</ul>
```

Comment:
Rather than introduce 10 new top-level config flags that we know we want to break, I'd rather we implement the social hash described in #146, and have a fallback to retain support for the old `foo_username` syntax for the two existing ones.